### ISSUE TITLE
UI: Managing virtual real estate in skore

### ISSUE THREAD

**Issue by anasstarfaoui** [probabl-ai/skore#543]
**Description:** 

Users are experiencing challenges in managing the virtual real estate within Skore, resulting in cluttered workspaces and difficulties navigating between different elements. We need to research and explore potential solutions to help users optimize their views space and improve the overall display of items.

**User Story:** 

As a Skore user, I want more efficient ways to optimize the display of items on my view, so I can work more productively and have a clearer overview of my projects.

**Acceptance Criteria:**

- [ ] Conduct user research to identify specific pain points in managing elements in views
- [ ] Explore and document potential solutions for optimizing views usage

**Comment by anasstarfaoui** [comment]
@tuscland

**Comment by anasstarfaoui** [comment]
https://www.figma.com/proto/Qzjyxv0lPDU2kKtsH9dcp3/Skore---Workstation---High-fidelity-mock-ups---Q3?node-id=4305-13619&node-type=frame&t=aMGCs4hFxJ6pjNeq-1&scaling=min-zoom&content-scaling=fixed&page-id=2073%3A6331&starting-point-node-id=2073%3A6338

Here is a [[link](https://www.figma.com/design/Qzjyxv0lPDU2kKtsH9dcp3/Skore---Workstation---High-fidelity-mock-ups---Q3?node-id=4305-12997&t=YRY4B68Wb9kYqCSB-4)](url) to a prototype to test a new version, more interesting from a information density display imo

**Comment by augustebaum** [comment]
I like it; the only thing I could think of though is that if there is more than one card, every time you hover, all the cards below will move to make space for the added text. This could get a bit tiring I would say.

**Comment by tuscland** [comment]
I agree with Auguste's feedback. Can we keep the height constant, and only display chrome when hovered?

Also, I believe one requirement was to avoid displaying the items' headers. To me this is a valid requirements in the scope of building a report.

Here, the idea is to start with a dashboard that is lean.

Can we optimize the header's height?

**Comment by tuscland** [comment]
Work done on the CrossValidationReporter shared some new insights and opportunities.

**Comment by anasstarfaoui** [comment]
![CrossValReporter](https://github.com/user-attachments/assets/3286aa15-191f-459b-981d-d0ccc4d757fd)
Yes, here is an alternative version, where we will not have the same issue as before, regarding element length because now everything is on the same line.

Also, we could think of contextual informations on hover.

What do you think?

**Comment by rouk1** [comment]
👍🏻 
Where should the label that says that your viewing an old version should go ?

**Comment by MarieSacksick** [comment]
We will remove the lib UI for at least the next 6 months; we can close this issue.
